README.md

```diff
@@ -43,7 +43,16 @@ This creates:
 * a Subcription to the OCS catalog in the `openshift-storage`
 namespace
 
-Once this is done, a StorageCluster can be created.
+You can check the status of CSV by command `oc get csv -n openshift-storage`
```

Comment:
Change the above to: `You can check the status of the CSV using the following command:`, and then include the command in the console output as Kaushal suggests.

Reply:
done